src/Symfony/Component/Form/FormBuilderInterface.php

```diff
@@ -38,7 +38,7 @@ public function add($child, $type = null, array $options = array());
      * @param string|FormTypeInterface $type    The type of the form or null if name is a property
      * @param array                    $options The options
      *
-     * @return FormBuilderInterface The created builder
+     * @return $this
```

Comment:
This change looks wrong to me. It doesn't return the current instance, but create another one, for a child for instance.